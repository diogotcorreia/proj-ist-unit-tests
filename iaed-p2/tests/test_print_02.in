set a 1
set b 1
set c 1
set d 1
set e 1
set /a/b/c 1
set /a/b 1
set /b/a 1
set /e/b 1
set /c/a/e 1
set /a/c/e 1
set /e/a/b 1
set /d/a/c 1
set c/b/d 1
set /a/b/c/d 1
set e/a/b/c 1
set e/b/c/d/e 1
set b/a/d/c/a 1
print
quit
