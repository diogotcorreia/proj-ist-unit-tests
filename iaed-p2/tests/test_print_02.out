/a 1
/a/b 1
/a/b/c 1
/a/b/c/d 1
/a/c/e 1
/b 1
/b/a 1
/b/a/d/c/a 1
/c 1
/c/a/e 1
/c/b/d 1
/d 1
/d/a/c 1
/e 1
/e/b 1
/e/b/c/d/e 1
/e/a/b 1
/e/a/b/c 1
